src/main/scala/io/github/pauljamescleary/petstore/infrastructure/repository/doobie/DoobieUserRepositoryInterpreter.scala

```diff
@@ -48,24 +47,26 @@ class DoobieUserRepositoryInterpreter[F[_]: Monad](val xa: Transactor[F])
 
   import UserSQL._
 
-  def create(user: User): F[User] =
+  def put(user: User): F[User] =
```

Comment:
funny how we are revisiting this.  I guess this is ok, as we still fail if we violate the primary key constraint

Reply:
I forgot this part, I think PR #79 had allowed this change to creep in, but this is surely out of scope here, I'll revert it